src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -684,6 +692,9 @@ private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, Dat
                     throw Host.ExceptSchemaMismatch(nameof(inputSchema), "label", (string)labelColumn.Name, "Key", (string)labelType.ToString());
 
                 _classCount = labelCount == 1 ? 2 : (int)labelCount;
```

Comment:
>labelCount == 1 ? 2 : (int)labelCount [](start = 30, length = 37)

Why?
/cc @codemzs

Reply:
To account for binary classification case where in the train set all you have seen is just one class :-). 

---
In reply to: [327483161](https://github.com/dotnet/machinelearning/pull/4228#discussion_r327483161) [](ancestors = 327483161)